corgidrp/l2a_to_l2b.py

```diff
@@ -88,6 +88,35 @@ def convert_to_electrons(input_dataset):
 
     return None 
 
+def em_gain_division(input_dataset):
+    """
+    
+    Convert the data from electrons to photo-electrons by dividing the commanded em_gain. 
+    update the change in units in the header [e/phot].
+
+    Args:
+        input_dataset (corgidrp.data.Dataset): a dataset of Images (L2a-level)
+
+    Returns:
+        corgidrp.data.Dataset: a version of the input dataset with the data in units photo-electrons
+    """
+    
+    # you should make a copy the dataset to start
+    emgain_dataset = input_dataset.copy()
+
+    emgain = emgain_dataset[0].ext_hdr["CMDGAIN"]
+    emgain_cube = emgain_dataset.all_data/emgain
+    emgain_error = emgain_dataset.all_err/emgain
```

Comment:
I'm not sure why the error is divided by the gain.  The error is the uncertainty in knowledge of the signal photo-electrons?

Reply:
Yes, I think the error should always be relative to the signal, so have the same units, also the different kind of errors.